07-gitops.md

```diff
@@ -1,106 +0,0 @@
-# Place the Cluster Under GitOps Management
```

Comment:
Please let's ensure all narratives from here are moved over the cluster deployment section

Reply:
done